Misc/NEWS.d/next/Windows/2025-05-20-21-43-20.gh-issue-130727.-69t4D.rst

```diff
@@ -0,0 +1,2 @@
+Fix a race in :mod:`_wmi` that can result in an "invalid handle"
```

Comment:
Just call it "internal calls into WMI". It's not a public API, but we aren't hiding that we use WMI for some things.